docs/contributing/code-documentation.txt

```diff
@@ -48,33 +48,30 @@ Structure
     # @return [ Tiger ] The transmogrified result.
     def transmogrify(person)
 
+- **Errors:** Use ``@raise`` to explain errors specific to the method.
+
+  .. code-block:: ruby
+
+    # @raise [ Errors::Validations ] If validation failed.
+    def validate!
```

Comment:
Can you please put everything that is not related to yielding to another PR.

Reply:
Extracted here: https://github.com/mongodb/mongoid/pull/5429